src/app/store.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
+
+import {jsonStoreFileName} from './constants';
+
+type AppStore = {
+    bot_access_token: string;
+    oauth2_client_secret: string;
+}
+
+class Store {
+    store: AppStore;
```

Comment:
Maybe call this `storeData` to avoid confusion?

Reply:
done.

Reply:
https://github.com/mattermost/mattermost-app-zendesk/commit/fa8a3e3c7bcebec08dd689fbae3927ceb05f7847